src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/CollectionInstrumentSvcClient.java

```diff
@@ -24,5 +87,23 @@
    * @return count of collection instruments matching the search string
    * @throws RestClientException something went wrong making http call
    */
-  Integer countCollectionInstruments(String searchString) throws RestClientException;
+  public Integer countCollectionInstruments(String searchString) throws RestClientException {
+    MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
+    queryParams.add("searchString", searchString);
+
+    UriComponents uriComponents =
+        restUtility.createUriComponents(
+            appConfig.getCollectionInstrumentSvc().getRequestCollectionInstrumentsCount(),
+            queryParams);
+
+    HttpEntity<List<CollectionInstrumentDTO>> httpEntity = restUtility.createHttpEntity(null);
+
+    ResponseEntity<String> responseEntity =
+        restTemplate.exchange(uriComponents.toUri(), HttpMethod.GET, httpEntity, String.class);
```

Comment:
Seriously? We have a REST API endpoint that returns a non-rest value (i.e. a number with no JSON wrapping). We should change the API at the other end to wrap the response in JSON so it's RESTful.